src/components/store/actions/blockReducerActions.ts

```diff
@@ -0,0 +1,33 @@
+import { OutputBlockData } from '../../../../types';
+import { BlockMutationType } from '../../../../types/events/block/mutation-type';
+import { ChangeBlockDataAction, CreateBlockAction, RemoveBlockAction } from '../../../../types/store/action';
+
+/**
+ * Action creator for creating a new block in the store
+ *
+ * @param block - new block data
+ */
+export const createBlock = (block: OutputBlockData): CreateBlockAction => ({
```

Comment:
why anonymous function? Its better to create regular named function:

```ts
function createBlock(block: OutputBlockData): CreateBlockAction {
  return {
    // ...
  }
}

```

Reply:
and probably it would better to name it `makeCreateBlockAction` because that is what this function does.